meetings/2025-01-29.md

```diff
@@ -0,0 +1,45 @@
+# Node.js  TypeScript team Meeting 2025-01-29
+
+## Links
+
+* **Recording**: <https://www.youtube.com/watch?v=9Wa0FBTxFN8>
+* **GitHub Issue**: <https://github.com/nodejs/typescript/issues/26>
+
+## Present
+
+* Typescript team: @nodejs/typescript
+* Marco Ippolito: @marco-ippolito
+* Jake Bailey: @jakebailey
+* Lucas Santos: @khaosdoctor
+* Augustin Mauroy @AugustinMauroy
+
+## Agenda
+
+## Announcements
+
+Jake: Typescript 5.8 --erasableSyntaxOnly out, Node.js needs to do some changes in the swc implementation to match the ts behavior on namespaces, test cases can be copied from <https://github.com/bloomberg/ts-blank-space/pull/32>.
+
+* Extracted from **typescript-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/typescript
+
+* Roadmap to stable strip-types [#24](https://github.com/nodejs/typescript/issues/24)
+Marco:
+* working improving errors, currently blocked by swc.
+* need to create a tsconfig for users
+
+* Jake: probably better to add it to <https://github.com/tsconfig/bases> instead of creating a CLI tool.
+    > Lucas volunteered (wait until 5.8 is out on beta and state that it will only work for the beta in the description).
```

Comment:
```suggestion
  * Lucas volunteered (wait until 5.8 is out on beta and state that it will only work for the beta in the description).
```

If you look at the markdown preview, this causes the list to render with double spacing, which is also a markdown oddity that pops up.

Reply:
Hah, I thought that'd fix it, but it didn't. :((